flow/raster_cache.h

```diff
@@ -100,6 +100,8 @@ class RasterCache {
 
   void SetCheckboardCacheImages(bool checkerboard);
 
+  size_t size() const;
```

Comment:
Please follow the style guide for function names. go/fluttercpp#Function_Names. `GetCachedEntriesCount` might make more sense to the casual reader.

Reply:
Changed. I was thinking of both of these methods as getters so lowercase but the longer function names are probably more clear.